code/modules/projectiles/energy_bolt.dm

```diff
@@ -7,9 +7,11 @@
 //How much ammo this costs
 	cost = 15
 //How fast the power goes away
-	dissipation_rate = 2
+	dissipation_rate = 1
 //How many tiles till it starts to lose power
 	dissipation_delay = 2
+//how many tiles (not quite tiles) the projectile can go regardless of falloff
```

Comment:
If you could put the comments here after the variables, that would be great.
Also, clarification on `how many tiles (not quite tiles)`?

Reply:
I'll add a comment for clarification about how falloff doesn't quite seem to work by tiles to the base projectile datum. Essentially, for a straight shot (entirely vertical or entirely horizontal), the projectile tickers doesn't quite line up to 1 tick = 1 tile at default projectile speeds in my testing, leading to 'double' falloff at certain distances.